### DISCUSSION TITLE
Account - Expire Time

### DISCUSSION THREAD

**Discussion by Ox29** [Ylianst/MeshCentral#5787] [Q&A] [answered]
It's possible to enable expire time to account?
example, i have employees outside the company and need to limit account access by period of time, expire the account after 3 months

**Comment by si458** [comment, marked as answer]
No this isn't possible at all sorry
The best u can do is 'lock' the account, but this would have to do done manually via ldap/webui or using the meshctrl.js via an automation

**Comment by Ox29** [comment]
ok, thank you

Maybe someday this will be possible?

**Reply by si458** [reply]
Create a github issue (enhancement) and I can look into it in my free time
